### PR TITLE
fix: chrome router hijack rug

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

_Actually_ fixes the router rug on clicking trade preview.

tl;dr, chrome thinks the form submission is a GET request if the `onSubmit` isn't wrapped in it's special helper.

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A, but fixes what https://github.com/shapeshift/web/pull/4947 tried to fix.

## Risk

Small.

## Testing

With the multi-hop flag on, select a non-fee EVM asset, get a quote, and click "Preview Trade". It should no lnoger change routes and re-render.

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4950** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4950" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4950?utm_source=stack-comment).

**Review by reallybeard — APPROVED** [review]
